src/puppetlabs/puppetdb/cli/import.clj

```diff
@@ -96,7 +96,7 @@
                              options)
         construct-base-url (fn [{:keys [host port] :as options}]
                              (-> options
-                                 (assoc :base-url (utils/pdb-admin-base-url host port export/admin-api-version))
+                                 (assoc :base-url (utils/pdb-admin-base-url host port :v1))
```

Comment:
I can change it back, I removed it because I didn't know exactly where it should live, probably `admin.clj` wouldn't be a bad place for that though.